### PR TITLE
logs: Add audit logs for organization and user namespace activities (PROJQUAY-3482)

### PR DESCRIPTION
**Pull request by dmesser**
This adds several new action log event types, entries and corresponding UI support for the following activities:

- user/organization creation
- user/organization deletion
- user enable / disable via superuser
- organization rename via superuser
- change of an user/organization's email address, invoicing and time machine settings
- change of a users's password
- user creation via registration
- user Docker CLI password generation

Changes have been tested and verified with the table-based log model as well as the log list view UI.

Screenshot 2023-04-28 at 10 45 28
Screenshot 2023-04-28 at 12 20 53
Screenshot 2023-04-29 at 22 12 11

### CODE REVIEW

**Comment by dmesser** [comment]
No idea what that failing cyprus test in the am64 build is about, but I suppose it is unrelated?

**Comment by dmage** [comment]
SNAFU, failing tests are not related to this PR.

File: endpoints/api/superuser.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 from flask import request, make_response, jsonify
 
 from cryptography.hazmat.primitives import serialization
+import auth
```
Comment (dmage) [inline endpoints/api/superuser.py:15]: Is this related to your changes?

**Review by dmage — DISMISSED** [review]

File: endpoints/api/superuser.py (hunk 1 above)
Comment (dmesser) [reply endpoints/api/superuser.py:15]: Looks like this is not needed.

**Review by dmage — APPROVED** [review]
